### PR TITLE
feat(fscomponents): Implement SingleLineForm with Formik

### PR DESCRIPTION
**Pull request by Bodrus**
Notes:

Implemented Formik and Yup for SingleLineFormFK

### CODE REVIEW

**Comment by Cauldrath** [comment]
Please include a story.tsx file for this component.

**Comment by Cauldrath** [comment]
Please include a story.tsx file for this component.

**Comment by Bodrus** [comment]
@Cauldrath Story is added

**Comment by Cauldrath** [comment]
Conflict.

**Comment by Bodrus** [comment]
@Cauldrath Conflicts are resolved

File: packages/fscomponents/src/components/SingleLineFormFK.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  const labelPosition = (typeof props.labelPosition === 'number') ?
+    props.labelPosition : FormLabelPosition.Inline;
+
+  const onSubmit = (values: FormValues) => {
+    if (values && props.onSubmit) {
+      props.onSubmit(values);
+    }
+  };
+
+  const handleBoton = (f: FormikProps<FormValues>) => () => {
```
Comment (Cauldrath) [inline packages/fscomponents/src/components/SingleLineFormFK.tsx]: "handleBoton" - I'm guessing this should be handleButton?

File: packages/fscomponents/src/components/SingleLineFormFK.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  labelStyle?: StyleProp<TextStyle>;
+  containerStyle?: StyleProp<ViewStyle>;
+  labelPosition?: FormLabelPosition;
+  validationSchema?: any | (() => any);
+  fieldsOptions?: FieldOption;
+  placeholder: string;
+  fieldsStyleConfig?: any;
+}
+
+const bootonStyle = StyleSheet.create({
```
Comment (Cauldrath) [inline packages/fscomponents/src/components/SingleLineFormFK.tsx]: buttonStyle

**Comment by Bodrus** [comment]
@Cauldrath Fixed. Please look.

**Pull request merged**